wsd/RequestVettingStation.cpp

```diff
@@ -298,27 +302,16 @@ void RequestVettingStation::checkFileInfo(const Poco::URI& uri, bool isReadOnly,
             const auto docKey = RequestDetails::getDocKey(uriPublic);
             LOG_DBG("WOPI::CheckFileInfo succeeded and will create DocBroker ["
                     << docKey << "] now with URL: [" << url << ']');
-            if (_ws)
+
+            if (createDocBroker(docKey, url, uriPublic))
             {
-                if (createDocBroker(docKey, url, uriPublic))
+                assert(_docBroker && "Must have docBroker");
+                if (_ws)
                 {
-                    assert(_docBroker && "Must have docBroker");
+                    // If we don't have the WebSocket, deffer creating the client session.
                     createClientSession(docKey, url, uriPublic, isReadOnly);
                 }
             }
```

Comment:
Good to have a similar erro LOG_DBG as below here to describe the 'else' state here for !ws ?